src/components/Contact/Contact.tsx

```diff
@@ -27,6 +27,7 @@ type ContactProps = {
   institutions?: InstitutionProps[];
   contactRole?: string;
   name?: string;
+  nested?: string;
```

Comment:
Should this be a boolean?

If so it might make more sense for it to be `isNested`